lib/utils/bpr.js

```diff
@@ -11,16 +16,34 @@ function _getConditions(node) {
   _getCallee(node.test, callees);
 
   if (node.test.left) {
+    // If the global reference is part of the conditional and is on the left side (i.e. it's before the
+    // isBrowser check), return an empty array of callees to make isEnvironmentBrowser/isBrowser check false.
+    if (isGlobalPartofCondition(node, globalReferenceNode) && node.test.left.object === globalReferenceNode) {
+      return [];
+    }
     _getCallee(node.test.left, callees);
   }
 
   if (node.test.right) {
+    // If the global reference is part of the conditional and is on the right side (i.e. it's after the
+    // isBrowser check), and the operator is ||, return an empty array of callees to make
```

Comment:
Same as above. If we can refactor this to remove the comment that would be ideal.